### PR TITLE
Switched from `expansion_strategy` to `level` in Fourier tutorial

### PR DESCRIPTION
**Pull request by Shiro-Raven**

### CODE REVIEW

File: demonstrations/tutorial_expressivity_fourier_series.py [hunk 1]
```diff
@@ -198,160 +198,167 @@ def square_loss(targets, predictions):
...
 
 ######################################################################
 # We now define the quantum model itself.
-# 
+#
 
 scaling = 1
 
-dev = qml.device('default.qubit', wires=1)
+dev = qml.device("default.qubit", wires=1)
```
Comment (isaacdevlugt) [inline demonstrations/tutorial_expressivity_fourier_series.py:277]:
```suggestion
dev = qml.device("default.qubit")
```

File: demonstrations/tutorial_expressivity_fourier_series.py [hunk 2]
```diff
@@ -450,102 +457,106 @@ def cost(weights, x, y):
...
 # Let's have a quick look at the ansatz itself for 3 qubits by making a
 # dummy circuit of 2 ansatz layers:
-# 
+#
 
 n_ansatz_layers = 2
 n_qubits = 3
 
-dev = qml.device('default.qubit', wires=4)
+dev = qml.device("default.qubit", wires=4)
```
Comment (isaacdevlugt) [inline demonstrations/tutorial_expressivity_fourier_series.py:497]:
```suggestion
dev = qml.device("default.qubit")
```

File: demonstrations/tutorial_expressivity_fourier_series.py [hunk 3]
```diff
@@ -450,102 +457,106 @@ def cost(weights, x, y):
...
 ######################################################################
 # Now we define the actual quantum model.
-# 
+#
 
 scaling = 1
 r = 3
 
-dev = qml.device('default.qubit', wires=r)
+dev = qml.device("default.qubit", wires=r)
```
Comment (isaacdevlugt) [inline demonstrations/tutorial_expressivity_fourier_series.py:517]:
```suggestion
dev = qml.device("default.qubit")
```

**Review by isaacdevlugt — APPROVED** [review]
Awesome!

**Review by Alex-Preciado — APPROVED** [review]
Looks good!! 😎

**Pull request merged**